spec/openapi.yaml

```diff
@@ -1,50 +1,245 @@
+                $ref: '#/components/schemas/Category_data'
+              example: |
+                {
+                  'apiVersion':'1.0',
+                  'data':{
+                    'id':1,
+                    'name':'Books',
+                  },
+                  'status':'ok',
+                  'status_code':200
+                }
+        404:
+          $ref: '#/components/responses/NotFound'
   /languages:
     get:
-      description: Gets list of all languages and topics available
+      summary: List Languages
+      description: Gets list of all languages and topics available such as Python, SQL, and Artificial Intelligence
+      tags:
+        - List Languages
       responses:
         200:
+          description: A list of languages
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/inline_response_200'
-              examples:
-                0:
-                  value: |
-                    {"apiVersion":"1.0","data":[{"id":1,"name":"multiple"},{"id":2,"name":"Python"},{"id":3,"name":"Unix/Linux/*nix"},{"id":4,"name":"Bash"},{"id":5,"name":"C"},{"id":6,"name":"Sysadmin"},{"id":7,"name":"JavaScript"},{"id":8,"name":"HTML"},{"id":9,"name":"PHP"},{"id":10,"name":"Ruby"},{"id":11,"name":"Aspectwerkz"},{"id":12,"name":"Artificial Intelligence"},{"id":13,"name":"C++"},{"id":14,"name":"MongoDB"},{"id":15,"name":"SQL"},{"id":16,"name":"R"},{"id":17,"name":"GraphQL"},{"id":18,"name":"Swift"},{"id":19,"name":"Android"},{"id":20,"name":"CSS"}],"has_next":true,"has_prev":false,"number_of_pages":4,"page":1,"records_per_page":20,"status":"ok","status_code":200,"total_count":73}
-  /resources/{id}:
+                $ref: '#/components/schemas/Language'
+              example: |
+                    {
+                      'apiVersion':'1.0',
+                      'data':[
+                        {'id':1,'name':'multiple'},
+                        {'id':2,'name':'Python'},
+                        {'id':3,'name':'Unix/Linux/*nix'},
+                        {'id':4,'name':'Bash'},
+                        {'id':5,'name':'C'},
+                        {'id':6,'name':'Sysadmin'},
+                        {'id':7,'name':'JavaScript'},
+                        {'id':8,'name':'HTML'},
+                        {'id':9,'name':'PHP'},
+                        {'id':10,'name':'Ruby'},
+                        {'id':11,'name':'Aspectwerkz'},
+                        {'id':12,'name':'Artificial Intelligence'},{'id':13,'name':'C++'},
```

Comment:
I'm not sure why these are on the same line, but it bothers me lol. Can we move resource 13 to its own line please? Thanks 😄

Reply:
Ooh good catch! Missed that line when I was changing the spacing.